### PR TITLE
add tooltips to imatrics tags

### PR DESCRIPTION
**Pull request by petrjasek**
SDNTB-692

### CODE REVIEW

File: scripts/extensions/auto-tagging-widget/src/types.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ export interface ITagUi {
     description?: string;
     qcode: string;
     source?: string;
+    original_source?: string;
+    aliases?: Array<string>;
```
Comment (tomaskikutis) [inline scripts/extensions/auto-tagging-widget/src/types.ts:7]: Could you add comments on what those are?

File: scripts/extensions/auto-tagging-widget/src/tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,11 @@
 {
     "extends": "superdesk-code-style/tsconfig-strict",
     "compilerOptions": {
-        "outDir": "../dist/src"
-    }
+        "outDir": "../dist/src",
+        "skipLibCheck": true,
```
Comment (tomaskikutis) [inline scripts/extensions/auto-tagging-widget/src/tsconfig.json:5]: What about installing [types](https://github.com/react-bootstrap/react-bootstrap-typescript)? If that library doesn't work, you could try importing it as `any` so it doesn't affect other libraries.

```
import __OverlayTrigger from 'react-bootstrap/OverlayTrigger';

const OverlayTrigger = __OverlayTrigger as unknown as any;
```

File: scripts/extensions/auto-tagging-widget/src/types.ts (hunk 1 above)
Comment (petrjasek) [reply scripts/extensions/auto-tagging-widget/src/types.ts:7]: it's actually aliases :wink: so other names which are used for the concept

File: scripts/extensions/auto-tagging-widget/src/tsconfig.json (hunk 2 above)
Comment (petrjasek) [reply scripts/extensions/auto-tagging-widget/src/tsconfig.json:5]: will revert those changes, I had some issues but that got fixed when updating typescript

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**